src/dripper/polkadot/PolkadotActions.ts

```diff
@@ -98,46 +98,44 @@ export class PolkadotActions {
   async teleportTokens(dripAmount: bigint, address: string, parachain_id: string): Promise<DripResponse> {
     logger.info("💸 teleporting tokens");
 
-    const prefix = networkName === "rococo" ? "Staging" : "";
-
-    const dest = polkadotApi.createType(`${prefix}XcmVersionedMultiLocation`, {
-      V3: polkadotApi.createType(`${prefix}XcmV3MultiLocation`, {
-        interior: polkadotApi.createType(`${prefix}XcmV3Junctions`, {
-          X1: polkadotApi.createType(`${prefix}XcmV3Junction`, {
+    const dest = {
+      V3: {
+        interior: {
+          X1: {
             Parachain: polkadotApi.createType("Compact<u32>", parachain_id),
```

Comment:
But do we still need this one?

Reply:
yea, i just tested - it also infers this, so removed